lib/BloqadeExpr/src/interface.jl

```diff
@@ -5,6 +5,16 @@ const n = YaoBlocks.ConstGate.P1
 
 end
 
+function process_atom_positions(atom_positions)
+    if typeof(atom_positions) <: BoundedLattice
+        return atoms
```

Comment:
```suggestion
        return atom_positions
```
Assume it should be this because `RydInteract` now accepts `BoundedLattice` as a type for `atoms`